### PR TITLE
Update aws cli to v2

### PR DESCRIPTION
**Pull request by rezabekf**
## What does this PR do and why?

Add instructions and script to migrate AWS CLI v1 to AWS CLI v2.

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [ ] Added/Updated documentation if applicable.
- [x] Pre-commit checks passed.
- [x] Lint and Nag checks passed.
- [ ] If releasing a new version, have you bumped the version `make version part=<major|minor|patch>`?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/prerequisites/cloud9/index.md [hunk 1]
```diff
@@ -29,6 +29,34 @@ We recommend using **us-east-1 (N. Virginia)** as the _AWS Region_ for the works
 If you have issues using the Cloud9 environment, please see the [Troubleshooting AWS Cloud9](https://docs.aws.amazon.com/cloud9/latest/user-guide/troubleshooting.html) page in the user guide.
 :::
 
+### Install the latest version of AWS CLI
+
+Cloud9 Instance comes pre-installed with AWS CLI version one. For the workshop we require to use AWS CLI version 2.
```
Comment (rezabekf) [inline content/prerequisites/cloud9/index.md:34]:
```suggestion
Cloud9 Instance comes pre-installed with AWS CLI version 1. For the workshop we require to use AWS CLI version 2.
```

**Comment by rezabekf** [comment]
FYI, the script only assumes that awscli v1 is installed via `yum` which is the case at the moment. I thought it would be overkill to check for all the possible ways to install awscli...it can be pip, brew and what not...possibly we could add the official docs links at the end for people to install manually if our way fails?

File: content/prerequisites/cloud9/index.md [hunk 2]
```diff
@@ -29,6 +29,34 @@ We recommend using **us-east-1 (N. Virginia)** as the _AWS Region_ for the works
 If you have issues using the Cloud9 environment, please see the [Troubleshooting AWS Cloud9](https://docs.aws.amazon.com/cloud9/latest/user-guide/troubleshooting.html) page in the user guide.
 :::
 
+### Install the latest version of AWS CLI
+
+Cloud9 Instance comes pre-installed with [AWS CLI version 1](https://docs.aws.amazon.com/cli/v1/userguide/install-linux-al2017.html). For the workshop we require to use [AWS CLI version 2](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+To make the [migration](https://docs.aws.amazon.com/cli/latest/userguide/cliv2-migration-instructions.html) from version 1 to version 2 easier you can use the script provided.
```
Comment (rezabekf) [inline content/prerequisites/cloud9/index.md:35]:
```suggestion
To make the [migration](https://docs.aws.amazon.com/cli/latest/userguide/cliv2-migration-instructions.html) from version 1 to version 2 easier, you can use the script provided.
```

File: content/prerequisites/cloud9/index.md [hunk 3]
```diff
@@ -29,6 +29,34 @@ We recommend using **us-east-1 (N. Virginia)** as the _AWS Region_ for the works
...
+1. Run the script:
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+source awscliv2.sh
+:::
+1. Check that the installation was successful, and you have the latest version of AWS CLI:
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+aws --version
+:::
+
 ### Clone lab resources using `git`
```
Comment (cjhar) [inline content/prerequisites/cloud9/index.md:60]: This needs to come before the "Install the latest version of AWS CLI" section on line 32

**Review by cjhar — COMMENTED** [review]
Thanks Franco, other than the order this tested okay on a fresh Cloud9 instance in us-east-1

File: content/prerequisites/cloud9/index.md (hunk 3 above)
Comment (rezabekf) [reply content/prerequisites/cloud9/index.md:60]: oh yeah, of course haha

**Review by sdangol — APPROVED** [review]
LGTM

**Pull request merged**